app/Widgets/Twitter/Commands/TwitterContent.php

```
@@ -0,0 +1,130 @@
+<?php
+/**
+ * This file is part of Digital Atlas.
+ *
+ * Digital Atlas is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Digital Atlas is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @license http://opensource.org/licenses/gpl-license.php GNU Public License
+ *
+ */
+namespace App\Widgets\Twitter\Commands;
+
+use App\Widgets\Twitter\Models\Twitter;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use TwitterAPIExchange;
+
+/**
+ * Command for widget.
+ */
+class TwitterContent extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     * @access protected
+     */
+    protected $signature = 'import:twitter:data';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     * @access protected
+     */
+    protected $description = 'Imports Twitter data from the API.';
+
+    protected $twitter_api = '';
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     * @access public
+     */
+    public function __construct()
+    {
+        $apiKey = [
```

Can you move these settings into the configuration file?  I wrote new documentation on it [here](https://github.com/MobMin/digital_atlas/wiki/Widgets#widget-configuration).  Please throw an error if none of the configurations are set.